### DISCUSSION TITLE
使用群晖chat推送

### DISCUSSION THREAD

**Discussion by bzmshang** [RayWangQvQ/BiliBiliToolPro#676] [General]
请教下，我该在这个括号中写什么参数，才能推送出每日任务的日志
![image](https://github.com/RayWangQvQ/BiliBiliToolPro/assets/111360926/cb869f17-0887-4ff9-98be-7e8b8ec40f7a)

**Comment by SoilGuo** [comment]
https://github.com/RayWangQvQ/BiliBiliToolPro/blob/main/src/Ray.BiliBiliTool.Console/appsettings.json
https://kb.synology.com/zh-hk/DSM/help/Chat/chat_integration?version=7#b_5

参考源码注释和synology文档：
Ray_Serilog__WriteTo__8__Args__api: 填 webhook url
Ray_Serilog__WriteTo__8__Args__placeholder :#msg#
Ray_Serilog__WriteTo__8__Args__bodyJsonTemplate :payload={"text": #msg#}

**Reply by bzmshang** [reply]
非常感谢，可以使用了

**Reply by shange1993** [reply]
> 非常感谢，可以用了

请问下具体如何设置呢，我不会设置
